pkg/featureflag/featureflag.go

```diff
@@ -55,6 +55,9 @@ var EnableExternalDNS = New("EnableExternalDNS", Bool(false))
 // SpecOverrideFlag allows setting spec values on create
 var SpecOverrideFlag = New("SpecOverrideFlag", Bool(false))
 
+// EnableEtcdEncryption if will enable encryption-at-rest for etcd.
+var EnableEtcdEncryption = New("EnableDataEncryption", Bool(false))
```

Comment:
dependent on @justinsb and @chrislovecnm if a feature flag is required

Reply:
We can pull it